docs/config.md

```diff
@@ -225,13 +222,14 @@ def setup_test_database():
     """
     Create a clean test database every time the tests are run.
     """
-    url = str(settings.DATABASE_URL)
-    engine = create_engine(url)
-    assert not database_exists(url), 'Test database already exists. Aborting tests.'
-    create_database(url)             # Create the test database.
-    metadata.create_all(engine)      # Create the tables.
-    yield                            # Run the tests.
-    drop_database(url)               # Drop the test database.
+    engine = create_engine(settings.DATABASE_URL)
```

Comment:
What was wrong with this?

Reply:
we had to do `url = str(settings.DATABASE_URL)` to convert DatabaseURL from databases to a string for SQLAlchemy create_engine.
That's not needed anymore, we can pass around the string DB URL.

Reply:
it was still more cute before